src/cljs/main/broadfcui/common.cljs

```diff
@@ -196,11 +196,11 @@
   (or (access-equal-to? level1 level2) (access-greater-than? level1 level2)))
 
 (defn count-workflows [entity root-entity-type]
-  (let [entity-type (entity "entityType")]
+  (let [entity-type (:entityType entity)]
     (cond (= entity-type root-entity-type) 1
           ;; example: entity is 'sample_set', RET is 'sample', presumably using expression 'this.samples'
           (= entity-type (singular-type->set-type root-entity-type))
-          (count (get-in entity ["attributes" (set-type->membership-attribute entity-type)]))
+          (count (get-in entity [:attributes (keyword (set-type->membership-attribute entity-type)) :items]))
```

Comment:
Explanation for this line:

Here is an example of a sample set entity

`{:name "sample_set2",
 :entityType "sample_set",
 :attributes
 {:samples
  {:itemsType "EntityReference",
   :items
   [{:entityType "sample", :entityName "HCC1143"}
    {:entityType "sample", :entityName "HCC1143_2"}
    {:entityType "sample", :entityName "HCC1143_3"}
    {:entityType "sample", :entityName "HCC1143_4"}
    {:entityType "sample", :entityName "HCC1143_5"}
    {:entityType "sample", :entityName "HCC1143_6"}
    {:entityType "sample", :entityName "HCC1143_7"}
    {:entityType "sample", :entityName "HCC1143_8"}
    {:entityType "sample", :entityName "HCC1143_9"}
    {:entityType "sample", :entityName "HCC1143_10"}]}}}
`

If our root entity type is sample, we resort to the second option on the cond. The line counts the number of items in the entity.  So we go into `:attributes`, and then `:samples` (that's what `(keyword (set-type->membership-attribute entity-type))` simplifies into. And then finally we go into `:items`